crates/sui-framework/sources/governance/validator_set.move

```diff
@@ -42,6 +45,11 @@ module sui::validator_set {
 
         /// Mappings from staking pool's ID to the sui address of a validator.
         staking_pool_mappings: Table<ID, address>,
+
+        /// Mapping from a staking pool ID to the inactive validator that has that pool as its staking pool.
+        /// When a validator is deactivated the validator is removed from `active_validators` it
+        /// is added to this table so that delegators can continue to withdraw their stake from it.
+        inactive_validators: Table<ID, Validator>,
```

Comment:
What is the rational behind keeping the entire `Validator` in this map, instead of just inactive_staking_pools where we only store the staking_pool?

Reply:
See the discussion here https://github.com/MystenLabs/sui/pull/8768#discussion_r1123602678